api/app/helpers/spree/api/api_helpers.rb

```diff
@@ -83,7 +83,7 @@ def required_fields_for(model)
 
       @@payment_attributes = [
         :id, :source_type, :source_id, :amount, :display_amount,
-        :payment_method_id, :response_code, :state, :avs_response, :created_at,
```

Comment:
In checkouts_controller_decorator.rb we set the :response_code attribute (with the braintree payment token), so I believe we might need to remove this change here.

If this change is absolutely necessary, let me know and I will concat this attribute in the `Spree::Api::CheckoutsController#set_strong_parameters` method in dotandbo-spree.